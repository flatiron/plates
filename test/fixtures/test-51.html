<a class="replace" data-bind="test">link</a>
