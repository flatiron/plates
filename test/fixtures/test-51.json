{
  "one": "Super link",
  "replaced": "replaced"
}
