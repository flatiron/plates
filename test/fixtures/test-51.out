<a class="replaced" data-bind="test">Super link</a>
